tests/support/cluster_util.tcl

```diff
@@ -256,7 +256,8 @@ proc start_cluster {masters replicas options code {slot_allocator continuous_slo
 
     # Configure the starting of multiple servers. Set cluster node timeout
     # aggressively since many tests depend on ping/pong messages. 
-    set cluster_options [list overrides [list cluster-enabled yes cluster-ping-interval 100 cluster-node-timeout 3000 cluster-slot-stats-enabled yes]]
+
+    set cluster_options [list overrides [list cluster-enabled yes cluster-ping-interval 100 cluster-node-timeout 3000 cluster-databases 16 cluster-slot-stats-enabled yes]]
```

Comment:
A few lines below this we set `::singledb` to 1. Normally we test on a random DB, which has helped us catch some random regressions in the past, I think it would be helpful if we can remove that to improve coverage. I'm okay skipping that for now, but would like to follow up and add it.

Reply:
https://github.com/valkey-io/valkey/issues/2021